docker-compose.solr.yaml

```diff
@@ -2,7 +2,7 @@
 
 services:
   solr:
-    image: solr:8
+    image: solr:9.6
```

Comment:
Is there a reason for using "9.6" instead of letting it float with "9" ? As you know, the dockerhub setup allows both. https://hub.docker.com/_/solr

Reply:
yes. During my tests, I got 9.2 installed, just because I had that image in a cache from another project. But the techproducts example is from 9.6.
I found no way to force a minimum version in docker-compose, something like >=9.6.0